astropy/io/fits/setup_package.py

```diff
@@ -18,11 +18,12 @@ def _get_compression_extension():
 
     cfg = defaultdict(list)
     cfg['include_dirs'].append(numpy.get_include())
-    cfg['sources'].append(os.path.join(os.path.dirname(__file__), 'src', 'compressionmodule.c'))
+    cfg['sources'].append(os.path.join(os.path.dirname(__file__),
```

Comment:
nitpicking, but this was OK. At the coordination meeting (or around it), we decided to go with 100 as line length.